frontend/src/templates/Header/Header.tsx

```diff
@@ -1,5 +1,79 @@
-
+import { Box, AppBar, Toolbar, IconButton, Typography, Avatar, Button } from "@mui/material";
+import {ReactComponent as Logo} from "../../assets/logo.svg";
+import {SvgIcon} from "@mui/material";
+import { useTheme } from "@mui/material";
 export function Header()
 {
-    
-}
\ No newline at end of file
+  const them = useTheme();
+    return(
+    <Box
+    maxHeight={100}>
+      <AppBar
+      >
+        <Toolbar>
+          <Box
+          flexGrow={1}  
+          display="flex"     
+          >
+        <IconButton
+            size='small'
+            edge='start'
+          >
+        <Logo
+    height={100}
+    width={400}
+    fill="white"
+    />
+            </IconButton>
+            </Box>
+            <Button variant="text" color="inherit"
+          >
+            <Typography
+            variant='h6'
+            component='div'
+            fontFamily={'Roboto'}
+          >
+            CONTACT US
```

Comment:
![image](https://user-images.githubusercontent.com/72614768/224573725-e59b5aae-6252-4959-9c08-54d041b58ec7.png)
This does not render as Roboto
![image](https://user-images.githubusercontent.com/72614768/224573740-73d97a47-7eb6-43cc-84e2-4fba8ad3d6d6.png)